administration/jsondb.md

```diff
@@ -30,3 +45,158 @@ Example file -:
 ```
 
 ```
```

Comment:
Do we need this empty example file code block or does the new content make this obsolete?